File: src/main/java/com/mateusz/rest/webservices/restful_web_services/jpa/UserRepository.java
package com.mateusz.rest.webservices.restful_web_services.jpa;

import org.springframework.data.jpa.repository.JpaRepository;

import com.mateusz.rest.webservices.restful_web_services.users.User;

public interface UserRepository extends JpaRepository<User, Integer> {

}

File: src/main/kotlin/com/mateusz/rest/webservices/restful_web_services/spring_boot_social_media_restful_api/jpa/UserRepository.kt
package com.mateusz.rest.webservices.restful_web_services.spring_boot_social_media_restful_api.jpa

import com.mateusz.rest.webservices.restful_web_services.spring_boot_social_media_restful_api.users.User
import org.springframework.data.jpa.repository.JpaRepository

interface UserRepository : JpaRepository<User?, Int?>
